### PR TITLE
feat(icons): SVG sprites from figma

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**

We don't use icons from Figma yet

**What is the chosen solution to this problem?**

Storybook replaces our old demo page.  

Icons from Figma are now part of SVG sprites split by size.  

- http://4041.talend.surge.sh/icons/dist/svg-bundle/8.svg
- http://4041.talend.surge.sh/icons/dist/svg-bundle/12.svg
- http://4041.talend.surge.sh/icons/dist/svg-bundle/16.svg
- http://4041.talend.surge.sh/icons/dist/svg-bundle/24.svg

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/design-system/src/components/Icon/Icon.tsx [hunk 1]
```diff
@@ -130,7 +130,7 @@ export const Icon = React.forwardRef(
 				// eslint-disable-next-line no-param-reassign
 				current.innerHTML = content;
 			} else if (current && !isRemote) {
-				IconsProvider.injectIcon(name, current);
+				IconsProvider.injectIcon(name.split('talend-').reverse()[0] + ':16', current);
```
Comment (frassinier) [inline packages/design-system/src/components/Icon/Icon.tsx:133]:
```suggestion
				IconsProvider.injectIcon(name.split('talend-').reverse()[0] + '\\:16', current);
```

**Comment by frassinier** [comment]
wip

**Review by jmfrancois — COMMENTED** [review]
it would be nice to align columns
Screenshot 2022-05-25 at 17 15 17

and may be make it more condensed and ready to use using the name:size pattern if you want for example

**Review by Benrajalu — APPROVED** [review]

**Review by jmfrancois — APPROVED** [review]
* please remove the H2 in the storybook it is just repetitions of the icon name and it is too verbose.
* move the psotinstall into build:lib

**Pull request merged**